node/config.go

```diff
@@ -385,7 +385,13 @@ func (c *Config) NodeKey() *ecdsa.PrivateKey {
 		return key
 	}
 
-	keyfile := c.ResolvePath(datadirPrivateKey)
+	// Updating default path to nodekey to be $Chain/nodekey
+	var keyfile string
+	if filepath.IsAbs(datadirPrivateKey) {
```

Comment:
Do we need to handle absolute paths for when we're setting `keyfile` for the `SaveECDSA` code below?